server/parsec/components/auth.py

```diff
@@ -155,6 +162,77 @@ def verify_signature(self, verify_key: VerifyKey) -> bool:
             return False
 
 
+class AccountAuthenticationType(enum.Enum):
+    Password = b"password"
+
+
+@dataclass
+class AccountAuthenticationToken:
+    auth_type: AccountAuthenticationType
+    timestamp: DateTime
+    email: EmailStr
+    header_and_payload: tuple[bytes, ...]
+    signature: bytes
+
+    HEADER = b"PARSEC-HMAC-BLAKE2B"
+
+    @staticmethod
```

Comment:
`AuthenticatedToken` has this comment for `generate_raw`: `# Only used for tests, but coherent to have it here`

Should keep the comment if it's still the case